### PR TITLE
Eliminate use of SqlQueryText()

### PR DESCRIPTION
**Pull request by klightspeed**

### CODE REVIEW

**Comment by robbyxp1** [comment]
Your way too fast for me ;-) ta

File: EliteDangerous/DB/BookmarkClass.cs [hunk 1]
```diff
@@ -372,27 +372,30 @@ public static bool LoadBookmarks()
...
                         {
-                            BookmarkClass bc = new BookmarkClass(dr);
-                            gbl.globalbookmarks.Add(bc);
+                            bookmarks.Add(new BookmarkClass(rdr));
                         }
+                    }
+                }
 
-                        return true;
+                if (bookmarks.Count != 0)
```
Comment (Esvandiary) [inline EliteDangerous/DB/BookmarkClass.cs]: Should this be `== 0`?

Comment (robbyxp1) [reply EliteDangerous/DB/BookmarkClass.cs]: i'll check when i review, ta

**Comment by robbyxp1** [comment]
Thanks

**Pull request merged**